tests/__init__.py

```diff
@@ -84,6 +111,81 @@ def setup_package():
     """
     dj.config['safemode'] = False
 
+    # Add old MySQL
+    source = "/src/tests/external-legacy-data"
+    db_name = "djtest_blob_migrate"
+    db_file = "v0_11.sql"
+    conn_root.query("""
+        CREATE DATABASE {};
+        """.format(db_name))
+
+    def parse_sql(filename):
+        data = open(filename, 'r').readlines()
+        stmts = []
+        DELIMITER = ';'
+        stmt = ''
+
+        for lineno, line in enumerate(data):
+            if not line.strip():
```

Comment:
strip spaces from the line for the following checks too.

Reply:
Hi Dimitri, shouldn't there be spaces between the command and arguments? If I also strip the spaces won't it cause the following `USE djtest_blob_migrate;\n` to become `USEdjtest_blob_migrate;`?

Reply:
I see. You are right. But for example, the next line `line.startswith('--')` will fail if there is a leading zero.

Reply:
Instead of adding up the strings you can be joining on `" "`.